src/gui/Connected.qml

```diff
@@ -651,30 +689,6 @@ Item {
                         color: textColour
                     }
 
-                    Text {
```

Comment:
FYI I removed this because the text is pretty lengthy and pushes things around in the pop-up. But this message appears in the connected view already